server/scripts/feed-data.ts

```diff
@@ -26,17 +26,18 @@ import { collections } from '../db/schema';
     // all responses to inserts, including duplicates and errors
     const response = [];
     // for debugging: the last record attempted to process before error
-    /**
-     * @type {any}
-     */
-    let lastInsert = '';
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let lastInsert: any = '';
```

Comment:
This is odd, I assume based off the explicit exception above that this needs to be any.

Reply:
Yes, because it could be a string, or an arbitrary object of unknown value. We don't care about its properties, since it's just for printing anyways, so `any` it is.